conda_smithy/ci_register.py

```diff
@@ -50,9 +50,7 @@
     anaconda_token = os.environ["BINSTAR_TOKEN"]
 except KeyError:
     try:
-        with open(
-            os.path.expanduser("~/.conda-smithy/anaconda.token"), "r"
-        ) as fh:
```

Comment:
Last change was by black in https://github.com/conda-forge/conda-smithy/commit/b4ea9c39676644ff83945c0a84ff91ea76f490a7 .
Why did black decide to do this earlier and now is reverting it?

Reply:
I might have put in the pep8 line length flag in last time 🐑

Reply:
Ok added that flag to CI and re-ran with it. Also squashed all black commits to avoid adding noise to the history.